tests/WebAssertTest.php

```diff
@@ -992,33 +994,67 @@ public function testFieldValueNotEquals()
         ;
 
         $this->session
-            ->expects($this->exactly(4))
+            ->expects($this->any())
             ->method('getPage')
             ->will($this->returnValue($page))
         ;
 
         $page
-            ->expects($this->exactly(4))
+            ->expects($this->any())
             ->method('findField')
-            ->with('username')
+            ->with('test')
             ->will($this->returnValue($element))
         ;
 
         $element
-            ->expects($this->exactly(4))
+            ->expects($this->any())
             ->method('getValue')
-            ->will($this->returnValue(235))
+            ->will($this->returnValue($fieldValue))
         ;
 
-        $this->assertCorrectAssertion('fieldValueNotEquals', array('username', 234));
         $this->assertWrongAssertion(
             'fieldValueNotEquals',
-            array('username', 235),
+            array('test', $fieldValue),
             'Behat\\Mink\\Exception\\ExpectationException',
-            'The field "username" value is "235", but it should not be.'
+            'The field "test" value is "'.
+            (is_array($fieldValue) ? '['.implode(', ', $fieldValue).']' : $fieldValue)
+            .'", but it should not be.'
         );
-        $this->assertCorrectAssertion('fieldValueNotEquals', array('username', 23));
-        $this->assertCorrectAssertion('fieldValueNotEquals', array('username', ''));
+
+        foreach ($errors as $value) {
+            $this->assertCorrectAssertion('fieldValueNotEquals', array('test', $value));
+
+        }
+    }
+
+    public function getFieldValues()
+    {
+        return [
+            'empty string' => [
```

Comment:
Please use only PHP 5.3 compatible syntax, because of PHP version requirements in `composer.json`.